rabbitmq/datadog_checks/rabbitmq/openmetrics/metrics.py

```diff
@@ -0,0 +1,228 @@
+    "erlang_vm_atom_limit": "erlang.vm.atom_limit",
+    "erlang_vm_dirty_cpu_schedulers": "erlang.vm.dirty_cpu_schedulers",
+    "erlang_vm_dirty_cpu_schedulers_online": "erlang.vm.dirty_cpu_schedulers_online",
+    "erlang_vm_dirty_io_schedulers": "erlang.vm.dirty_io_schedulers",
+    "erlang_vm_dist_node_queue_size_bytes": "erlang.vm.dist.node.queue_bytes",
+    "erlang_vm_dist_node_state": "erlang.vm.dist.node.state",
+    "erlang_vm_dist_port_input_bytes": "erlang.vm.dist.port.input_bytes",
+    "erlang_vm_dist_port_memory_bytes": "erlang.vm.dist.port.memory_bytes",
+    "erlang_vm_dist_port_output_bytes": "erlang.vm.dist.port.output_bytes",
+    "erlang_vm_dist_port_queue_size_bytes": "erlang.vm.dist.port.queue_bytes",
+    "erlang_vm_dist_proc_heap_size_words": "erlang.vm.dist.proc.heap_size_words",
+    "erlang_vm_dist_proc_memory_bytes": "erlang.vm.dist.proc.memory_bytes",
+    "erlang_vm_dist_proc_message_queue_len": "erlang.vm.dist.proc.message_queue_len",
+    "erlang_vm_dist_proc_min_bin_vheap_size_words": "erlang.vm.dist.proc.min_bin_vheap_size_words",
+    "erlang_vm_dist_proc_min_heap_size_words": "erlang.vm.dist.proc.min_heap_size_words",
+    "erlang_vm_dist_proc_reductions": "erlang.vm.dist.proc.reductions",
+    "erlang_vm_dist_proc_stack_size_words": "erlang.vm.dist.proc.stack_size_words",
+    "erlang_vm_dist_proc_status": "erlang.vm.dist.proc.status",
+    "erlang_vm_dist_proc_total_heap_size_words": "erlang.vm.dist.proc.total_heap_size_words",
+    "erlang_vm_dist_recv_avg_bytes": "erlang.vm.dist.recv.avg_bytes",
+    "erlang_vm_dist_recv_bytes": "erlang.vm.dist.recv.bytes",
+    "erlang_vm_dist_recv_cnt": "erlang.vm.dist.recv.cnt",
+    "erlang_vm_dist_recv_dvi_bytes": "erlang.vm.dist.recv.dvi_bytes",
+    "erlang_vm_dist_recv_max_bytes": "erlang.vm.dist.recv.max_bytes",
+    "erlang_vm_dist_send_avg_bytes": "erlang.vm.dist.send.avg_bytes",
+    "erlang_vm_dist_send_bytes": "erlang.vm.dist.send.bytes",
+    "erlang_vm_dist_send_cnt": "erlang.vm.dist.send.cnt",
+    "erlang_vm_dist_send_max_bytes": "erlang.vm.dist.send.max_bytes",
+    "erlang_vm_dist_send_pend_bytes": "erlang.vm.dist.send.pend_bytes",
+    "erlang_vm_ets_limit": "erlang.vm.ets_limit",
+    "erlang_vm_logical_processors": "erlang.vm.logical_processors",
+    "erlang_vm_logical_processors_available": "erlang.vm.logical_processors.available",
+    "erlang_vm_logical_processors_online": "erlang.vm.logical_processors.online",
+    "erlang_vm_memory_atom_bytes_total": "erlang.vm.memory.atom_bytes",
+    "erlang_vm_memory_bytes_total": "erlang.vm.memory.bytes",
+    "erlang_vm_memory_dets_tables": "erlang.vm.memory.dets_tables",
+    "erlang_vm_memory_ets_tables": "erlang.vm.memory.ets_tables",
+    "erlang_vm_memory_processes_bytes_total": "erlang.vm.memory.processes_bytes",
+    "erlang_vm_memory_system_bytes_total": "erlang.vm.memory.system_bytes",
+    "erlang_vm_msacc_alloc_seconds_total": "erlang.vm.msacc.alloc_seconds",
```

Comment:
When collecting metrics from counters, we have to leave out the `_total`. 

```suggestion
    "erlang_vm_msacc_alloc_seconds": "erlang.vm.msacc.alloc_seconds",
```

Reply:
Which could answer why this and the other counters are exposed in the test fixture and this mapping and isn't getting collected.

Reply:
This is *exactly* what's happening and is addressed in the next PR, thanks for the pointer!